docs/src/main/sphinx/admin/fault-tolerant-execution.md

```diff
@@ -145,6 +149,14 @@ with a `TASK` retry policy for large batch queries, separate from another
 cluster that handles short queries.
 :::
 
+(fte-encryption)=
+## Encryption
+
+Trino encrypts data before spooling it to storage. This prevents access to query data
+by anyone besides the Trino cluster that wrote it, including administrators of the
+storage system. A new encryption key is randomly generated for every query, and keys
```

Comment:
Technically it is for every exchagne within a query - but probably we may skip details.

Reply:
https://github.com/trinodb/trino/pull/20333